### PR TITLE
Upgrade fre:ac from 1.1-alpha-20190423 to 1.1-alpha-20190423a

### PR DESCRIPTION
**Pull request by photomoose**
Fixes compatibility issue with macOS Catalina.

### CODE REVIEW

File: Casks/freac.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'freac' do
-  version '1.1-alpha-20190423'
-  sha256 'ce10882ca82198ec84397ab3a2d1fb3292dd9272ab59db2e22226cf09cab2739'
+  version '1.1-alpha-20190423a'
+  sha256 'f6cbe820d0c8c114064b1282ea45e256a95df0c4433483043477cfe77c3627f5'
 
   # github.com/enzo1982/freac was verified as official when first introduced to the cask
-  url "https://github.com/enzo1982/freac/releases/download/v#{version}/freac-#{version}-macosx.dmg"
```
Comment (reitermarkus) [inline Casks/freac.rb]: This still works.

Comment (photomoose) [reply Casks/freac.rb]: @reitermarkus No it doesn't; there's no trailing 'a' in the version after /download.

Comment (reitermarkus) [reply Casks/freac.rb]: Oh, right. Maybe use `.sub(/a$/, '')` on the first interpolation then.

Comment (photomoose) [reply Casks/freac.rb]: @reitermarkus I've replaced the interpolation as suggested.

**Pull request merged**